rsync/entrypoint.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Auth
+echo "$INPUT_CLOUD_CREDENTIALS" > /secrets.json
+gcloud auth activate-service-account --key-file=/secrets.json
+rm /secrets.json
+
+# Check destination
+if [[ $INPUT_TO =~ $INPUT_CHECK_DESTINATION ]]; then
+    echo "Destination exists"
+else
+    echo "Error - Destination doesn't exist"
```

Comment:
Change to "Error - Destination not allowed" and append the allowed destinations to the error